src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
+                        "quotactl\0"
+                        "reboot\0"
+                        "seccomp\0"
+                        "setdomainname\0"
+                        "setfsgid32\0"
+                        "setfsgid\0"
+                        "setfsuid32\0"
+                        "setfsuid\0"
+                        "setgid32\0"
+                        "setgid\0"
+                        "setgroups32\0"
+                        "setgroups\0"
+                        "sethostname\0"
+                        "setregid32\0"
+                        "setregid\0"
+                        "setresgid32\0"
+                        "setresgid\0"
+                        "setresuid32\0"
+                        "setresuid\0"
+                        "setreuid32\0"
+                        "setreuid\0"
+                        "settimeofday\0"
+                        "setuid32\0"
+                        "setuid\0"
+                        "stime\0"
+                        "swapoff\0"
+                        "swapon\0"
+                        "sysctl\0"
+                        "syslog\0"
+                        "umount2\0"
+                        "umount\0"
+                        "vhangup\0",
+
+                        /* Network or Unix socket IO, should not be needed if not network facing */
+                        "@NetworkIO",
+                        "accept4\0"
+                        "accept\0"
+                        "bind\0"
+                        "connect\0"
+                        "getpeername\0"
+                        "getsockname\0"
+                        "getsockopt\0"
+                        "listen\0"
+                        "recv\0"
+                        "recvfrom\0"
+                        "recvmmsg\0"
+                        "recvmsg\0"
+                        "send\0"
+                        "sendmmsg\0"
+                        "sendmsg\0"
+                        "sendto\0"
+                        "setsockopt\0"
+                        "shutdown\0"
+                        "socket\0"
+                        "socketcall\0"
+                        "socketpair\0",
+
+                        /* Message queues or SYSV IPC: unusual */
+                        "@IPC",
+                        "ipc\0"
+                        "mq_getsetattr\0"
+                        "mq_notify\0"
+                        "mq_open\0"
+                        "mq_timedreceive\0"
+                        "mq_timedsend\0"
+                        "mq_unlink\0"
+                        "msgctl\0"
+                        "msgget\0"
+                        "msgrcv\0"
+                        "msgsnd\0"
+                        "semctl\0"
+                        "semget\0"
+                        "semop\0"
+                        "semtimedop\0"
+                        "shmat\0"
+                        "shmctl\0"
+                        "shmdt\0"
+                        "shmget\0",
```

Comment:
Of course, this is a bit of a misnomer, as most IPC actually takes place via AF_UNIX sockets, i.e. `@network-io`... but I figure it's OK anyway...